content/telegraf/v1.24/configure_plugins/custom_builder.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Reduce Telegraf binary size with customized agent
+description: Use the custom builder tool to reduce Telegraf binary disk space. 
+menu:
+  telegraf_1_24:
+
+    name: Reduce binary size
+    Parent: Configure plugins
+    weight: 20
+---
+Reduce Telegraf's overall memory and disk footprint with the custom builder. This tool allows you to select which plugins to use in the Telegraf binary based on your configuration file. 
+
+## Requirements
+
+- Ensure you've installed the current version of [Go](https://go.dev/). 
+- Create your Telegraf configuration file with the plugins you want to use. For details, see [Configuration options](/telegraf/v1.24/configuration/).
```

Comment:
should this be `[Configuration options](/telegraf/latest/configuration/)` so it doesn't backtrack to 1.24?

Reply:
Not here because this is already/only in 1.24, and we want it to go to the same current version even when we have 1.25+.